archimedes/src/app/component/program/program.component.ts

```diff
@@ -12,7 +12,7 @@ export class ProgramComponent implements OnInit {
 
   ngOnInit() {}
 
-  getUrl(program) {
+  getCatalogUrl(program) {
     if (program.catalog_url) {
       return program.catalog_url;
     } else if (program.profiles && program.profiles.length && program.profiles[0].url) {
```

Comment:
Is the `else if` logic still necessary here now that the primary profile URL and catalog URLs are printed separately in the single degree template?